### DISCUSSION TITLE
[ERR_INVALID_URL]: Invalid URL: execution error

### DISCUSSION THREAD

**Discussion by bajipogadadanda** [webdriverio/webdriverio#12445] [General]
I have configured WDIO fraemwork and its been wokring. Today I have configured in an another machine, started execution of a sample test case, but ended-up with below error. Its a weird error which I haven't see till now.
I defaulted service as chromedriver, and browser as chrome. But not sure why runner is invoking "geckodriver" node module and throwing this error.
Pls can you help me on it.
node:internal/errors:496
    ErrorCaptureStackTrace(err);
    ^
 
TypeError [ERR_INVALID_URL]: Invalid URL
    at new NodeError (node:internal/errors:405:5)
    at new URL (node:internal/url:637:13)
    at new HttpProxyAgent (C:\WDIOUiFramework\node_modules\geckodriver\node_modules\http-proxy-agent\dist\index.js:44:50)
    at file:///C:/WDIOUiFramework/node_modules/geckodriver/dist/install.js:23:23 {
  input: 'http_proxy',
  code: 'ERR_INVALID_URL'

**Comment by christian-bromann** [comment]
@Bapugit are you behind a coorperate proxy? It seems that WebdriverIO failed to download Geckodriver due to an invalid url.

**Comment by bajipogadadanda** [comment]
Thx!
Let me investigate on it.

**Reply by adnanerlansyah403** [reply]
Any updates on this issue?
